templates/chain.html

```diff
@@ -761,6 +769,18 @@ <h3 id="rules-name" style="margin-top:2px;text-align:center"></h3>
                             <td><p id="potential-name"></td>
                             <td style="width:10%"><button class="button-success potential-button" onclick="addLink($(this))">add</button></td>
                         </tr>
+                        <tr>
+                            <td style="width:10%"><b>technique:</b></td>
+                            <td><p id="potential-technique"></td>
+                        </tr>
+                        <tr>
+                            <td style="width:10%"><b>description:</b></td>
+                            <td><p id="potential-description"></td>
```

Comment:
looks like we already have command and description <td>'s in here, so these are adding duplicate entries.

Reply:
Not sure I follow, do you mean that we don't need the technique AND description to be there or that the description is a replacement for the technique name?